manager/integration/tests/test_system_backup_restore.py

```diff
@@ -112,6 +116,66 @@ def test_system_backup_and_restore_volume_with_data(client, volume_name, set_ran
 
     check_volume_data(restored_volume, data)
 
+@pytest.mark.system_backup_restore   # NOQA
+def test_system_backup_and_restore_volume_with_data_and_backingimage(client, volume_name, set_random_backupstore):  # NOQA
+    """
+    Scenario: test system backup and restore volume with data and backingimage
+
+    Issue: https://github.com/longhorn/longhorn/issues/5085
+
+    Given a backingimage
+    Given volume created with the backingimage
+    And system backup created
+    And system backup in state Ready
+    And volume deleted
+    And backingimage deleted
+
+    When restore system backup
+    Then system restore should be in state Completed
+
+    When wait for backingimage restoration to complete
+    When wait for volume restoration to complete
+    And volume detached
+
+    Then attach volume
+    And volume should be healthy
+    And volume data should exist
+    """
+
+    host_id = get_self_host_id()
+
+    create_backing_image_with_matching_url(
+        client, BACKING_IMAGE_NAME, BACKING_IMAGE_RAW_URL)
+
+    volume = create_and_check_volume(
+        client, volume_name, backing_image=BACKING_IMAGE_NAME)
+    volume.attach(hostId=host_id)
+    volume = wait_for_volume_healthy(client, volume_name)
+
+    system_backup_name = system_backup_random_name()
+    client.create_system_backup(Name=system_backup_name)
+
+    system_backup_wait_for_state("Ready", system_backup_name, client)
+
+    cleanup_volume(client, volume)
+    cleanup_all_backing_images(client)
+
+    system_restore_name = system_restore_random_name()
+    client.create_system_restore(Name=system_restore_name,
+                                 SystemBackup=system_backup_name)
+
+    system_restore_wait_for_state("Completed", system_restore_name, client)
+
+    backing_image = client.by_id_backing_image(BACKING_IMAGE_NAME)
+    assert backing_image.sourceType == BACKING_IMAGE_SOURCE_TYPE_RESTORE
+
+    restored_volume = client.by_id_volume(volume_name)
+    wait_for_volume_restoration_completed(client, volume_name)
+    wait_for_volume_detached(client, volume_name)
+
+    restored_volume.attach(hostId=host_id)
+    restored_volume = wait_for_volume_healthy(client, volume_name)
+
```

Comment:
Should also check the data?

Reply:
Thanks for the advice
I think no, we can focus on the volume backup/restore with BackingImage
And for the data consistency
1. Volume already has its own test above
2. BackingImage use checksum to verified the data during backup/restore. If it is not consistency, BackingImage will be failed and so is the test.

So I think we can remove the data test in this test case to make it simple.

Reply:
I see. Can you add a comment to ensure @longhorn/qa is informed that the data integrity testing is covered somewhere else by looking at the case?

Reply:
Sure!
Added.